app/css/error-list.css

```diff
@@ -0,0 +1,13 @@
+.ErrorList {
+    color: #e51e1e;
+    font-size: 14px;
+}
+.ErrorList > li {
+    margin-bottom: 6px;
+}
+.ErrorList > li:last-of-type {
+    margin-bottom: 10px;
+}
+input + .ErrorList > li {
```

Comment:
This is too complex of a selector. I'm having a hard time understanding how those margins work. I suggest using more meaningful class names and setting those inside `Utils.refresh_error_list` or in the HTML of the page.

Some name suggestions for the concepts we're trying to style: FormErrorList, FieldErrorList.